lib/fuschia/entities/nucleo.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Fuschia.PesquisadorTables.Nucleo do
```

Comment:
```suggestion
defmodule Fuschia.Entities.Nucleo do
```

O módulo pai das entidades é `Entities`